### PR TITLE
T/zip 64 download tokens

### PR DESCRIPTION
**Pull request by nataliemona**
Switch from using zak token to using download token.

### CODE REVIEW

File: functions/zoom-uploader.py [hunk 1]
```diff
@@ -373,8 +373,9 @@ def get_series_catalog(self):
         try:
             resp = oc_api_request("GET", endpoint)
         except requests.exceptions.HTTPError as e:
-            if e.response.status_code == "404":
+            if e.response.status_code == 404:
```
Comment (nataliemona) [inline functions/zoom-uploader.py:376]: Status code is an integer... not a string... whoops

File: functions/slack/handler.py [hunk 2]
```diff
@@ -263,7 +263,7 @@ def slack_api_request(endpoint):
         f"https://slack.com/api/{endpoint}",
         headers={"Authorization": f"Bearer {SLACK_API_TOKEN}"},
     )
-    r.raise_for_status
+    r.raise_for_status()
```
Comment (lbjay) [inline functions/slack/handler.py:266]: there ought to be a pylint warning for this exact bug

File: tests/conftest.py [hunk 3]
```diff
@@ -114,6 +115,8 @@ def _message_maker(frozen_time, payload):
             zip_id = payload["payload"]["zip_id"]
         else:
             zip_id = f"auto-ingest-{payload_obj['uuid']}"
+
+        on_demand_ingest = True if zoom_event == "on.demand.ingest" else False
```
Comment (lbjay) [inline tests/conftest.py]: Hmmm. Why not 
```
on_demand_ingest = zoom_event == "on.demand.ingest"
```
?

**Review by lbjay — CHANGES_REQUESTED** [review]
Just one small change

**Pull request merged**